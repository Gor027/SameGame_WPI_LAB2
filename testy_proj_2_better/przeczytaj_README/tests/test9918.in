P 9  P PP9 99P  P9 99 PP9 P9
 9  PPPPP 9      P9PP9P P99999P9P9P   9P 9 P  PP999P99P PP P 9999     9P  9 P99
P99 PPP 99  999P9 9 P9PP
P9PPP 9P9  P99P999PPP P9P9PPP9P99 PPPP9 9P99 PP9P
P P9P9P9  999P P P P9 9 
99P9 9    P 99 9P P  P9P9 999999PPPP P9 PP P 9 
99 P  99 P9 P999   P9 9P9 9      PPP9P9 PPP99 9 9 99PPPP9P9 9 P
 9P9P 9  P  9  9P9PP  9 9  9 P9 PP   PPP99P9 P999P  9
 9PPP9 PP9 PPP  9  P P   99P 99P9PPP 
 P PPP9P9999    P PP9P 9 9 P9PP9 PPP P 9 
PPP999  P 999 9 PP9 P9
