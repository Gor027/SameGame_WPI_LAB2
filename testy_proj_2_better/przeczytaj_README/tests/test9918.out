            9
            9   P  99 P
 9          9P  9 9PP P    9
P9       99999 P999PP9PP9 P9 9P  9P
P9    9 999P99 9PPPPP9PPP 9P 9P9PPPP P9  9 P
P9 9 9P99PP9PP9PP9PP9P99P9P999999PPP P9 9P 9  P 9
P9 9 999PP9P99P99PPP9P9P9999PP99PPPP P9PPP9P PPPP   9
9999 99P9999P999P9PP9P9P999P999PPPPPPP9PPPP9 999999PP9P PP    9
9999999PP9999P99PP9PP999999P9PP99PPP9PP999P99P999P999PPP9P9P99P999P9P99
